docs/books/computer-games/computer-battle-games/the_vital_message/the_vital_message.py

```diff
@@ -0,0 +1,46 @@
+# Import 'random' so we can generate random numbers.
+# The 'string' module gives us access to string-related utilities
+#The 'time' module helps us to wait for some time
```

Comment:
It looks like you've branched off your first submission; that makes this difficult to review, because there's *2* pieces of work in this PR.